app/src/main/java/org/wikipedia/csrf/CsrfTokenClient.kt

```diff
@@ -78,6 +78,48 @@ class CsrfTokenClient(private val loginWikiSite: WikiSite, private val numRetrie
             }
         }
 
+    suspend fun getToken(): String {
```

Comment:
For now, can this whole function be replaced with:
```
suspend fun getToken(): String {
    return token.blockingFirst()
}
```
?

Reply:
yes!